Jenkinsfile

```diff
@@ -134,9 +134,9 @@ pipeline {
                     script {
                         // If this build is not a pull request, run owasp scan on the distribution. Otherwise run incremental scan
                         if (env.CHANGE_ID == null) {
-                            sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:check dependency-check:aggregate -q -B -Powasp-dist -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                            sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:aggregate -q -B -pl !$DOCS -P !itests,owasp-dist $DISABLE_DOWNLOAD_PROGRESS_OPTS'
                         } else {
-                            sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:check -q -B -Powasp -DskipTests=true -DskipStatic=true -pl !$DOCS -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                            sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:aggregate -q -B -pl !$DOCS -P !itests $DISABLE_DOWNLOAD_PROGRESS_OPTS'
                         }
```

Comment:
✏️ Shouldn't we always scan the distribution (i.e. zip files), even if this is a PR build? Is this if/else really necessary if it's not scanning changed modules?

Reply:
Same answer as above: we were previously doing it this way - ignoring the distribution for PR builds

Reply:
My opinion is that we should always scan the zips. I think the only benefit to doing a different task during PR builds was to only scan changed files. Since the command in this if statement is no longer using gib, I think we should do the same, full scan (i.e. include zip files) on PR builds.

Reply:
> I think the only benefit to doing a different task during PR builds was to only scan changed files

Well, the owasp-dist profile wasn't being run on the PR builds before either, and I see no reason why it couldn't be. It seems like an intentional decision to not scan the distro for PRs. I guess the question is why...

A couple points:
1. The unique distro CVEs are not usually DDF's fault directly, but rather because it pulls in a bunch of upstream features from Karaf and other projects. I could see that info mattering sometimes for people putting up PRs (if they're messing with feature files), but in general those CVEs will be unrelated to their changes.
2. If we did scan the distro for PR builds, it's going to take a lot longer to get to the place where we can start failing builds on OWASP scans. There are _so_ many more CVEs in the distro and most aren't even stuff we use at runtime.

With that in mind, I think it might be better to start with ignoring the distro for PR builds until we cut down on the distro CVEs. It's in our interest to get to the place where we can fail builds again as quickly as possible.

Reply:
Yes, this seems like the original decision we had on PR builds. It is not the ideal state but probably the best balance of performance, maintenance on dependencies, and process at the moment.